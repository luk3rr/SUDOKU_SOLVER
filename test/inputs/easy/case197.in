740006058 800000090 000000036 010720003 060100000 000003900 130400002 007008000 900000060 
