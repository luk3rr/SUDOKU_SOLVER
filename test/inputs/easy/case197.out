Solving the following grid:
7 4 0 | 0 0 6 | 0 5 8 
8 0 0 | 0 0 0 | 0 9 0 
0 0 0 | 0 0 0 | 0 3 6 
------+-------+------
0 1 0 | 7 2 0 | 0 0 3 
0 6 0 | 1 0 0 | 0 0 0 
0 0 0 | 0 0 3 | 9 0 0 
------+-------+------
1 3 0 | 4 0 0 | 0 0 2 
0 0 7 | 0 0 8 | 0 0 0 
9 0 0 | 0 0 0 | 0 6 0 


Solution found :')

7 4 3 | 9 1 6 | 2 5 8 
8 5 6 | 3 4 2 | 7 9 1 
2 9 1 | 8 5 7 | 4 3 6 
------+-------+------
4 1 9 | 7 2 5 | 6 8 3 
3 6 8 | 1 9 4 | 5 2 7 
5 7 2 | 6 8 3 | 9 1 4 
------+-------+------
1 3 5 | 4 6 9 | 8 7 2 
6 2 7 | 5 3 8 | 1 4 9 
9 8 4 | 2 7 1 | 3 6 5 

Algorithm: BFS
Total time: 68 ms
Total expanded states: 10472
